### DISCUSSION TITLE
推出新版本

### DISCUSSION THREAD

**Discussion by XuanHanTan-Work** [actifyjs/actify#26] [General]
你好，我们三个月没在npm推出新版本了，Actify并且已经有许多变化。你认为我们应该推出新版本吗？

**Comment by lerte** [comment]
你说得对，我一直目标不清晰，一会用TailwindCSS，一会又去掉TailwindCSS。
一会想走headlessui路线，一会又不想。目前市面上有很多组件库。还有我一直在等react 19正式发布。

![image](https://github.com/user-attachments/assets/732dd976-18df-49d8-96ea-b06ed2a435a0)

在这段时间内，我一直在独立开发别的东西。[Taildoor](https://www.taildoor.com)，一个浏览器扩展。
还有一个客户端，基于Tauri+Shadcn+Hugo+Zola+TailwindCSS的一个静态网站打包器的客户端。
![image](https://github.com/user-attachments/assets/7d59e704-0e74-43d4-8f9d-cd9e800225e6)


你有好的Roadmap可以写出来，我开始的想法是做react版本的[vuetify](https://vuetifyjs.com)，[基于material 3规范](https://material-web.dev/)

**Reply by XuanHanTan-Work** [reply]
最近我有看一看Taildoor，让我们Tailwind用户省了很多编辑CSS测试的时间！

近年来市面上的确有很多新组件库，都有各自的优势。但是我个人认为我们应该保持开始的想法，并专注着加上更多M3部件。既然现在Tailwind与React还是一个非常广泛的配套，Actify也是React独一无二的M3组件库，如果没有大问题，我想我们不应该经常改变依赖的组件库，就为用户提供一个稳定并符合M3规范的组件库吧。
